hc/accounts/tests/test_profile.py

```diff
@@ -108,3 +123,20 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
```

Comment:
After reviewing the code, the user Charlie doesn't have team access enabled hence why the expected error code is a `403` because access is being denied. the test for successful adding of a team member is handled in `test_it_adds_team_member` and the expected status code is `200` since Alice has team access enabled.